src/patterns.json

```diff
@@ -11,7 +11,6 @@
   "(?<![hg]m)score",
   "@[a-z]",
   "\\(at\\)[a-z]",
-  "\\(github\\.com/",
```

Comment:
These changes seem out of scope for this fix?

Reply:
True, this is a side effect of my creating the naive pattern. I examined parts of the existing regular expression and replaced them with more efficient patterns that allow the same coverage.